Landing-Page/index.html

```diff
@@ -26,41 +26,88 @@ <h1><span class="light">Site </span>Name goes here</h1>
 	<section id="showcase">
 		<div class="container">
 			<h1>Hi, Want to know where you belong</h1>
-			<p><i>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor magna aliqua"</p></i>
+			<p><i>“It does not matter how slowly you go as long as you do not stop.” — Confucius</p></i>
 			  <button type="submit" class="button_1">Dive In</button>
 			
 		</div>
 		
 	</section>
 
+
 	<section id="boxes">
 		<div class="container">
-			<h2>Word of Encouragement</h2>
+			<h1>Most Popular Viewed</h1>
```

Comment:
Most Viewed Career Paths should be better. Also, Career is spelt wrongly in many places in the code. Please correct them